build.yaml

```diff
@@ -12,7 +12,7 @@ steps:
     script: |
       set -ex
       cd /io
-      mkdir repo
+      mkdir -p repo
```

Comment:
I can confirm that it still does happen. I suspect it's because the new container is mounting to the same location on the vm where the repo folder resides. So this issue only occurs if you re-run. Which we've had to do often when it fails part way through.